src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+    },
+    pickerColorButton: "Verander kleur",
+    loading: "Data lezen van toestel...",
+    logo: {
+      altText: "Chrysalis logo",
+    },
+  },
+  dialog: {
+    ok: "Ok",
+    cancel: "Annuleer",
+    close: "Sluiten",
+  },
+  app: {
+    menu: {
+      "focus-not-detected": "Welkom",
+      systemInfo: "Probleem melden",
+      editor: "Layout & Kleurmap Editor",
+      layoutEditor: "Layout Editor",
+      colormapEditor: "Kleurmap Editor",
+      macroEditor: "Macro Editor",
+      firmwareUpdate: "Firmware Update",
+      keyboardSettings: "Toetsenbord Instellingen",
+      preferences: "Voorkeuren",
+      selectAKeyboard: "Selecteer een toetsenbord",
+      layoutCard: "Layout Kaart",
+      selectAnotherKeyboard: "Selecteer nog een toetsenbord",
+      chat: "Discord chat",
+      feedback: "Verstuur feedback",
+      exit: "Afsluiten",
+      keyboardSection: "Toetsenbord",
+      chrysalisSection: "Chrysalis",
+      miscSection: "Diversen",
+      upgradeAvailable: "Een upgrade is beschikbaar!",
+    },
+    deviceMenu: {
+      Homepage: "Homepagina",
+      Forum: "Forum",
+      Chat: "Chat",
+    },
+    cancelPending: {
+      title: "Niet-opgeslagen veranderingen weggooien?",
+      content: `Je hebt niet-opgeslagen veranderingen. Indien je doorgaat zullen ze verloren gaan.`,
+    },
+    actionRequired: "Actie noodzakelijk",
+  },
+  editor: {
+    keyType: "Toets type",
+    keyCode: "Toetsencode",
+    legacy: {
+      migrate: "Migreren",
+      warning: `We hebben oude toetsen gevonden op de keymap die niet langer ondersteund worden. Om te migreren naar nieuwe codes, klik op de Migreren knop.`,
+    },
+    sharing: {
+      title: "Backup & Terugzetten",
+      loadFromLibrary: "Laden van Bibliotheek",
+      loadFromBackup: "Laden van backups",
+      loadFromFile: "Laden van bestand...",
+      exportToFile: "Exporteren naar bestand...",
+      import: "Importeren",
+      selectLoadFile: `Selecteer bestand vanwaar een layout geladen moet worden`,
+      selectExportFile: `Selecteer een bestand waarnaar de layout moet ge├½xporteerd worden`,
+      dialog: {
+        layoutFiles: "Layout bestanden",
+        allFiles: "Alle bestanden",
+      },
+      errors: {
+        unableToLoad: "Onmogelijk om layout te laden uit geselecteerd bestand.",
+        parseFail: "Parsen van layout data gefaald.",
+        invalidLayoutData: "Bestand bevatte geen geldige layout data.",
+        saveFail: "Fout tijdens opslaan van ge├½xporteerde layout.",
+      },
+      importConfirm: {
+        title: "Importeer de geselecteerde layout?",
+        contents: `Dit zal de huidige layout overschrijven, en alle niet-opgeslagen veranderingen zullen verloren gaan. Ben je zeker dat je wil verdergaan?`,
+      },
+    },
+    sidebar: {
+      custom: {
+        title: "Aangepaste toetsencode",
+        help: `Laat je een aangepaste toetsencode toewijzen aan een toets, of een code die op dit moment niet gekend is door Chrysalis.`,
```

Comment:
It might just be me, but I feel like "op dit moment niet gekend is" has more of a connotation that it's normal that Chrysalis doesn't know the code.
"op dit moment niet herkend wordt" feels to me like there's some problem in one of the algorithms of Chrysalis.